src/main/java/core/basesyntax/Polygon.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public abstract class Polygon extends Figure {

```

Comment:
Why we need this class?

Reply:
We don't need this claass. I experemented with diffrent structure. I forgot to delete it.